README.md

```diff
@@ -342,13 +342,33 @@ The minimal requirement target is an OpenGL 3.3 compatible GPU. Clang 9+ or GCC
 First step is to install packages mandatory for building. For example, with Clang-9 toolchain:
 
 ```
-sudo apt-get install libjavascriptcoregtk-4.0-dev libgl1-mesa-dev libcurl4-openssl-dev clang-9 libc++-9-dev libc++abi-9-dev lld-9 ninja-build
+sudo apt-get install libgl1-mesa-dev libcurl4-openssl-dev clang-9 libc++-9-dev libc++abi-9-dev lld-9 ninja-build
 ```
 
-Then targeting a Ninja make file:
+Depending on the JavaScript engine you wan't to use, you will have to install the package accordingly:
+
+For JavaScriptCore:
+
+```
+sudo apt-get install libjavascriptcoregtk-4.0-dev 
+```
+
+For V8:
+
+```
+sudo apt-get install libv8-dev 
+```
+
+Then targeting a Ninja make file for JavaScriptCore:
+
+```
+cmake -GNinja -DJSCORE_LIBRARY=/usr/lib/x86_64-linux-gnu/libjavascriptcoregtk-4.0.so  -DNAPI_JAVASCRIPT_ENGINE=JavaScriptCore ..
+```
+
+For V8:
 
 ```
-cmake -GNinja -DJSCORE_LIBRARY=/usr/lib/x86_64-linux-gnu/libjavascriptcoregtk-4.0.so ..
+cmake -GNinja -DNAPI_JAVASCRIPT_ENGINE=V8 ..
 ```
```

Comment:
Might be clearer if you bucket the build steps together per JS engine. This is somewhat hard to read IMO. You are also missing a `For JavaScriptCore:` in the second step but will be moot if we group by JS engine.

Reply:
Indeed. Fixed by splitting in 2 subsections.